### ISSUE TITLE
useful resource for finding supported programs/workaround

### ISSUE THREAD

**Issue by jedahan** [doron-cohen/antidot#57]
Just came across this wiki page, seems nice https://wiki.archlinux.org/index.php/XDG_Base_Directory#Supported

**Comment by doron-cohen** [comment]
Yes this is where it all began... I can also add to it this link: https://github.com/grawity/dotfiles/blob/master/.dotfiles.notes
